### PR TITLE
DROID-2146 Objects | Enhancement | Allow pinning and unpinning types in the new object-creation panel

### PR DESCRIPTION
**Pull request by uburoiubu**
---

- [ ] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: presentation/src/main/java/com/anytypeio/anytype/presentation/objects/SelectObjectTypeViewModel.kt [hunk 1]
```diff
@@ -142,6 +190,47 @@ class SelectObjectTypeViewModel(
         }
     }
 
+    fun onPinTypeClicked(typeView: SelectTypeView.Type) {
```
Comment (konstantiniiv) [inline presentation/src/main/java/com/anytypeio/anytype/presentation/objects/SelectObjectTypeViewModel.kt:193]: could you please add Timber here

File: presentation/src/main/java/com/anytypeio/anytype/presentation/objects/SelectObjectTypeViewModel.kt [hunk 2]
```diff
@@ -142,6 +190,47 @@ class SelectObjectTypeViewModel(
...
+                        space = SpaceId(id = space),
+                        types = pinned.toList()
+                    )
+                )
+            }
+        }
+    }
+
+    fun onUnpinTypeClicked(typeView: SelectTypeView.Type) {
+        val state = viewState.value
```
Comment (konstantiniiv) [inline presentation/src/main/java/com/anytypeio/anytype/presentation/objects/SelectObjectTypeViewModel.kt:215]: also here

**Review by konstantiniiv — APPROVED** [review]

File: presentation/src/main/java/com/anytypeio/anytype/presentation/objects/SelectObjectTypeViewModel.kt (hunk 1 above)
Comment (uburoiubu) [reply presentation/src/main/java/com/anytypeio/anytype/presentation/objects/SelectObjectTypeViewModel.kt:193]: done

File: presentation/src/main/java/com/anytypeio/anytype/presentation/objects/SelectObjectTypeViewModel.kt (hunk 2 above)
Comment (uburoiubu) [reply presentation/src/main/java/com/anytypeio/anytype/presentation/objects/SelectObjectTypeViewModel.kt:217]: done

**Pull request merged**